package.json

```diff
@@ -18,25 +18,38 @@
     "type": "git",
     "url": "git://github.com/shakiba/planck.js.git"
   },
-  "main": "lib/index.js",
-  "types": "index.d.ts",
+  "main": "dist/planck.js",
+  "types": "lib/index.d.ts",
```

Comment:
There is one problem with changing this path: `index.d.ts` starts with
```
export as namespace planck;
```
This allows the typings to reflect the case when planck is a global variable (`import "planck.js"` or setting "planck.js" in the types field of tsconfig would result in the planck object being defined in every module as global object)
Either we can move this line to "lib/index.d.ts" or we can cut this line entirely (I don't know if this is widely used as one has to also set "allowUmdGlobalAccess" in the tsconfig)

Reply:
Ah, this was by mistake